test/server/browser-provider-pool-test.js

```diff
@@ -0,0 +1,18 @@
+var expect               = require('chai').expect;
```

Comment:
Maybe let's move new tests to the [browser-provider-test.js](https://github.com/DevExpress/testcafe/blob/master/test/server/browser-provider-test.js)?